dist/tools/git/git-cache

```diff
@@ -28,6 +41,17 @@ update() {
     set +ex
 }
 
+is_cached() {
```

Comment:
It's probably better to check for the tag with `_check_commit`, rather than for the url (in case of pkg updates in PRs). We can also expose that function, similarly to what I did in a private commit in https://github.com/cgundogan/RIOT/commit/00ad9978b7bbfa9f215f2c8abbb1dd966f84f07d